hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileHandlers.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.file.impl.handlers;
+
+import static java.util.Objects.requireNonNull;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class FileHandlers {
+
+    private final FileAppendHandler fileAppendHandler;
+
+    private final FileCreateHandler fileCreateHandler;
+
+    private final FileDeleteHandler fileDeleteHandler;
+
+    private final FileGetContentsHandler fileGetContentsHandler;
+
+    private final FileGetInfoHandler fileGetInfoHandler;
+
+    private final FileSystemDeleteHandler fileSystemDeleteHandler;
+
+    private final FileSystemUndeleteHandler fileSystemUndeleteHandler;
+
+    private final FileUpdateHandler fileUpdateHandler;
```

Comment:
Somehow codecov complains about less coverage on this file

Reply:
mmh, no idea.

Reply:
Probably needs more tests for each class with its `ClassNameTest`